### DISCUSSION TITLE
Image/Font Adjustments

### DISCUSSION THREAD

**Discussion by rc999985** [IBM/mac-ibm-notifications#36] [Q&A] [answered]
Greetings,

Thank you for this tremendous work. I've been working with this for a couple days and have found it quite flexible. There are two things I am having a challenge figuring out:

1. When I use an image in the pop up ui, it appears that size is set. No matter how small I make the image, it still fills up a big portion of the window. Is there a way to control this?
2. Is there a way to adjust the font type and size in the popup ui notification?

Thank you

**Comment by SMartorelli** [comment, marked as answer]
Hi @rc999985, sorry for the delay but I completely missed this thread! 
1. The image accessory view tries by default to occupy the entire width of the pop-up area it has to show the image and so the image is scaled up or down subsequently. If you would like to have more customisation parameters for the image size please feel free to open a new feature request. We encourage you to participate to the development of this tool. In the meantime I can suggest you to place you image in an "invisible" frame that will fill the pop-up image area but making the trick to save you image size. The image accessory view accept .png files so it will keep the transparency.
2. The subtitle and the white box accessory view of the pop-up supports markdown language so you can customise a bit the appearance of you text. We're working on the HTML format support for these components in order to enhance the customisation options.

Hope these will answer to your doubts!
